src/components/react-json-schemaForm/GeometryField/MapInteraction.js

```diff
@@ -35,6 +36,9 @@ const MapInteraction = ({
   const { t } = useTranslation();
 
   const { settings } = useContext(CRUDContext);
+  const isRouting = useMemo(() => (
```

Comment:
Do you really need a use memo for a boolean ?

Reply:
I removed it 👍